src/platforms/browser/webgl/FontLoader.js

```diff
@@ -86,17 +86,38 @@
     this._document = contextDocument || document;
   }
 
-  FontLoader.useAdobeBlank = !isIE || ieVer >= 11.0;
-  FontLoader.useResizeEvent = isIE && ieVer < 11.0 && typeof document.attachEvent !== "undefined";
-  FontLoader.useIntervalChecking = window.opera || (isIE && ieVer < 11.0 && !FontLoader.useResizeEvent);
-  FontLoader.referenceText = " !\"\\#$%&'()*+,-./0123456789:;<=>?@ABCDEFGHIJKLMNOPQRSTUVWXYZ[]^_`abcdefghijklmnopqrstuvwxyz{|}~";
-  FontLoader.referenceFontFamilies = FontLoader.useAdobeBlank ? ["AdobeBlank"] : ["serif", "cursive"];
-    FontLoader.adobeBlankFontFaceStyleId = "fontLoaderAdobeBlankFontFace";
-    FontLoader.adobeBlankReferenceSize = null;
```

Comment:
I'm not sure if these changes help native at all. But it is still good because we can eventually move to a single hc branch for mobile and non mobile